labkey-client-api/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # The LabKey Remote API Library for Java - Change Log
 
+## version XX
```

Comment:
1.5.1 hasn't actually been released yet, so this new change should be listed as one of the changes for this next release.  
```suggestion
## version 1.5.1
*Released*: 6 July 2022
* Fix NPE when saving assay protocol with transform scripts
* Add derivationDataScope to PropertyDescriptor
```